pkg/resources/storage_integration.go

```diff
@@ -236,11 +236,6 @@ func UpdateStorageIntegration(data *schema.ResourceData, meta interface{}) error
 		stmt.SetString("COMMENT", data.Get("comment").(string))
 	}
 
-	if data.HasChange("type") {
```

Comment:
If the type can't be changed I think we also should indicate that on the [schema for this resource](https://github.com/chanzuckerberg/terraform-provider-snowflake/pull/326/files#diff-d93d3a19e447fb8fa7b3c85eba8698e994215c2a5b1d35f9e44299dc21189597R26) by setting `ForceNew: true`

Reply:
Done.